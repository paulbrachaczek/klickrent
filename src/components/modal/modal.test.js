import {shallow} from 'enzyme';
import toJson from "enzyme-to-json";
import React from 'react';
import Modal from './modal.component.jsx';

it("renders correctly", () => {
    const ModalComponent = shallow(<Modal />);
    expect(toJson(ModalComponent)).toMatchSnapshot();
});
  