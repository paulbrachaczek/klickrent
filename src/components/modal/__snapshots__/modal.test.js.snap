// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="m-modal"
>
  <div
    className="m-modal__container"
  >
    <header
      className="m-modal__header"
    >
      <p>
        Direct request
      </p>
      <Button
        isCloseButton={true}
        onClick={[Function]}
      >
        Ã—
      </Button>
    </header>
    <FormInput
      handleChange={[Function]}
      type="text"
    />
    <div
      className="m-modal__results"
    >
      <ul />
    </div>
  </div>
</div>
`;
